src/EditorFeatures/CSharpTest/UseNamedArguments/UseNamedArgumentsTests.cs

```diff
@@ -345,6 +345,23 @@ void M(Action arg1, int arg2)
 }");
         }
 
+        [WorkItem(18848, "https://github.com/dotnet/roslyn/issues/18848")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseNamedArguments)]
+        public async Task TestNotMissingWhenInsideSingleLineArgument4()
+        {
+            await TestWithCSharp7(
```

Comment:
💭 I maybe would have preferred the previous one using this form, and the new one being the C# 8 test. On the other hand, this form does make it clear what impact C# 8 had on the test itself so maybe this is better.